### DISCUSSION TITLE
Hey the fumadocs logo cannot be seen after bookmarking

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#977] [General]
![{EB3B44CA-074B-4F84-918A-DFADDCDE1C74}](https://github.com/user-attachments/assets/6926eab0-5a43-4152-b08e-af1024faca66)
 
I have bookmarked the fumadocs website but it's logo is not seen, I don't know what is the reason for this issue. Maybe the metadata and Icon

**Comment by fuma-nama** [comment]
Sorry for being late, yes it's fixed
